### ISSUE TITLE
Question: should `_var_name` violate snake case?

### ISSUE THREAD

**Issue by drewmullen** [terraform-linters/tflint#1326]
Currently prefixing a local with `_` violates snake case... I'm not a master of the definition of snake case but it seems common enough convention that I'd ask if this should truly violate:

```hcl
rule "terraform_naming_convention" {
  enabled = true
  format  = "snake_case"
}
```

```
Notice: local value name `_subnets` must match the following format: snake_case (terraform_naming_convention)

  on data.tf line 5:
   5:   _subnets = module.calculate_subnets.subnets_by_type

Reference: https://github.com/terraform-linters/tflint/blob/v0.34.0/docs/rules/terraform_naming_convention.md
```

**Comment by drewmullen** [comment]
thank you @bendrucker for indulging me and for mentioning that we can adjust to our own preferences
